packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -118,17 +155,29 @@ class BottomSheet extends Component {
 				onMoveShouldSetResponder={ panResponder.panHandlers.onMoveShouldSetResponder }
 				onMoveShouldSetResponderCapture={ panResponder.panHandlers.onMoveShouldSetResponderCapture }
 				onAccessibilityEscape={ this.props.onClose }
+				{ ...rest }
 			>
 				<KeyboardAvoidingView
 					behavior={ Platform.OS === 'ios' && 'padding' }
 					style={ { ...backgroundStyle, borderColor: 'rgba(0, 0, 0, 0.1)', ...style } }
 					keyboardVerticalOffset={ -this.state.safeAreaBottomInset }
 				>
 					<View style={ styles.dragIndicator } />
-					{ hideHeader && <View style={ styles.emptyHeaderSpace } /> }
 					{ ! hideHeader && getHeader() }
-					<View style={ [ styles.content, contentStyle ] }>{ this.props.children }</View>
-					<View style={ { height: this.state.safeAreaBottomInset } } />
+					<ScrollView
+						bounces={ this.state.bounces }
+						onScroll={ this.onScroll }
+						scrollEventThrottle={ 16 }
+						style={ { maxHeight: this.state.maxHeight } }
+						contentContainerStyle={ [
+							styles.content,
+							hideHeader && styles.emptyHeader,
+							{ paddingBottom: this.state.safeAreaBottomInset },
```

Comment:
This seems to not be doing much. It seems like the space between the last button and the home indicator is quite small. I tested removing/changing this line without any effect.

Again, I trust @iamthomasbishop so check if this extra space is really necessary.

Reply:
Fixed, it looks the same as before fix!